sdk/textanalytics/tests.yml

```diff
@@ -4,5 +4,9 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: textanalytics
-    Clouds: 'Preview'
-    UnsupportedClouds: 'China'
+    CloudConfig:
+      Public:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+      Canary:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        Location: 'centraluseuap'
```

Comment:
If we want it, I was thinking of adding a CanaryLocation parameter to avoid the need for this, but also wanted to avoid too much parameter bloat at the same time.

Reply:
Sorry Ben. I forgot about this PR before I went on vacation.
I will prefer that, so we don't have duplication of the other clouds or configuration in each library.
Do you envision that other clouds would like to have their specific Location too? or only applies for Canary

Reply:
Going to merge this PR as is to unblock the build.